commands/utils.py

```diff
@@ -67,6 +67,8 @@ async def deploy(
     production: bool,
     description: str,
     check_constraints: bool,
+    exclude_check_run_name: list[str],
+    exclude_check_run_conclusion: list[str],
```

Comment:
I wonder if parameter names should be in plural form, like `exclude_check_run_names` and `exclude_check_run_conclusions`.

Reply:
I can't do that because the `click` library passes flag values as a keyword argument with the same name as the flag name. And I can't use plural form for the flags because `click` doesn't support [multi value options](https://click.palletsprojects.com/en/8.0.x/options/#multi-value-options) with variable number of values.

Reply:
Shame on me. I don't know how to read the documentation. Just checked, it works. Code updated.